torrentinfo.py

```diff
@@ -3,36 +3,69 @@
 """
 
 from sopel import module
-import bleach
 import requests
+import lxml
 from lxml import etree
+import re
+import bleach
```

Comment:
No need to move `bleach` to the end of the list.

Reply:
I did remove it from the code while testing it (before commits), and then re-added, that's why it got moved.